src/transaction.ts

```diff
@@ -36,7 +36,7 @@ export class TransactionContextImpl<T extends UserDatabaseClient> extends Operon
     readonly functionID: number,
     operationName: string
   ) {
-    super(operationName, span, logger, workflowContext);
+    super(operationName, span, logger, workflowContext.applicationVersion, workflowContext);
```

Comment:
All context constructors pass the `applicationVersion` to the parent `OperonContext`